app/models/invitation.rb

```diff
@@ -0,0 +1,9 @@
+require 'tokenable'
```

Comment:
As long as the module is in a path which is autoloaded, you shouldn't need to require it yourself.